modular_skyrat/master_files/code/modules/paperwork/employment_contract.dm

```diff
@@ -0,0 +1,46 @@
+
+/obj/item/paper/work_contract
+	icon_state = "paper_words"
+	throw_range = 3
+	throw_speed = 3
+	item_flags = NOBLUDGEON
+	var/employee_name = ""
+
+/obj/item/paper/work_contract/Initialize(mapload, new_employee_name)
+	if(!new_employee_name)
+		return INITIALIZE_HINT_QDEL
+	AddElement(/datum/element/update_icon_blocker)
+	. = ..()
+	employee_name = new_employee_name
+	name = "paper- [employee_name] employment contract"
+	add_raw_text("<center>Conditions of Employment</center>\
+	<BR><BR><BR><BR>\
```

Comment:
This is really giving me a headache to read through, can you try and format it so it's a bit more spaced out?